rl/agents/ddpg.py

```diff
@@ -125,14 +125,14 @@ def clipped_error(y_true, y_pred):
         # Combine actor and critic so that we can get the policy gradient.
         # Assuming critic's state inputs are the same as actor's.
         combined_inputs = []
-        critic_inputs = []
+        actor_inputs = []
```

Comment:
I think it is better to name it as state input. That will be more general.

Reply:
Makes sense. Let me send another PR and close this one.